packages/ui/src/kit/VRadio/VRadio.vue

```diff
@@ -14,7 +14,10 @@
     >
       <slot />
     </span>
-    <tooltip-molecule :label="tooltipLabel" />
+    <tooltip-molecule
```

Comment:
you can wrap by div and remove `margin-top:2px` from styles
```
<div>
  <tooltip-molecule :label="tooltipLabel" />
</div>
```
because `tooltip-molecule` already have `display: inline-block` and radio label have `display: flex`, it crossing incorrectly.
but single `div` can fix it as wrap

Reply:
Nice hack! It is. In future, when we probably will have one organism with tooltip, I would to suggest discard `vertical-align: sub` usage and use flexes for tooltip align. Most modern and clean (no `sub` hack) usage.

@kazurus ?

Reply:
Done

Reply:
As I can see, we need using organism